operations/iuf/examples/iuf_run.md

```diff
@@ -1,7 +1,65 @@
 # `iuf run` Examples
 
-(`ncn-m001#`) << TODO >>
+**`NOTE`** Due the the large number of arguments available for `iuf run` invocations, the following examples omit certain arguments in order to focus on those of interest for each example.
+
+(`ncn-m001#`) Execute stages `process-media` through `deliver-product` for activity `admin-230126`.
+
+```bash
+iuf -a admin-230126 run -b process-media -e deliver-product
+```
+
+---
+
+(`ncn-m001#`) Execute stages `process-media` through `deliver-product` for activity `admin-230126` but skip the `pre-install-check` stage.
+
+```bash
+iuf -a admin-230126 run -b process-media -e deliver-product -s pre-install-check
+```
+
+---
+
+(`ncn-m001#`) Execute the `update-vcs-config` stage for activity `admin-230126`.
+
+```bash
+iuf -a admin-230126 run -r update-vcs-config
```

Comment:
update-vcs-config will require supplying the information contained in the sw recipe product_vars.yaml, aka recipe vars.  there are a variety of ways to effectively supply this information but a good pattern is to supply a path to the recipe vars AND your site vars config file

Reply:
Addressed in next commit.